.github/workflows/ci.yml

```diff
@@ -15,7 +15,6 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '0.7' 
```

Comment:
Okay I'm doing it.

Reply:
Could you then please bump the compat bound in Project.toml and also the latexify minor version?

Reply:
nah, not in that case